charts/argo-workflows/README.md

```diff
@@ -175,7 +181,7 @@ Fields to note:
 | server.serviceType | string | `"ClusterIP"` | Service type for server pods |
 | server.sso | object | `{}` | SSO configuration when SSO is specified as a server auth mode. |
 | server.tolerations | list | `[]` | [Tolerations] for use with node taints |
-| server.volumeMounts | list | `[]` | Additional volume mounts to the server main container. |
```

Comment:
Hmm. Is this change intentional?

Reply:
Wow, don't quite know what happened there, lemme fix